README.md

```diff
@@ -65,7 +65,7 @@ Installing Zim is easy. If you have a different shell framework installed (like
   source ${ZDOTDIR:-${HOME}}/.zlogin
   ```
 
-5. You're done! Enjoy your Zsh IMproved! Take some time to read about the [available modules][modules] and tweak your `.zshrc` file.
+5. You're done! Enjoy your Zsh IMproved! Take some time to read about the [available modules][modules] and tweak your `.zshrc` file. To set input mode, set `zinput_mode` in `.zimrc`.
```

Comment:
Not sure if this is so important to go to the main README.

Reply:
I do think it should be mentioned at all, since it's a deviation from pure zsh settings, but it's not important where.